tidoop-mr-lib-api/src/server.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-tidoop (FI-WARE project).
+ *
+ * fiware-tidoop is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-tidoop is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-tidoop. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with
+ * francisco dot romerobueno at telefonica dot com
+ */
+
+/**
+ * Http server for "Tidoop MR job library" REST API
+ *
+ * Author: frb
+ */
+
+var Hapi = require('hapi');
+
+// create a server with a host and port
+var server = new Hapi.Server();
+var config = require('../conf/tidoop-mr-lib-api.json');
+
+server.connection({ 
+    host: 'localhost',
+    port: config.port
+});
+
+// add the route
+server.route({
+    method: 'GET',
+    path:'/version',
+    handler: function (request, reply) {
+        console.log("Request: GET /version")
```

Comment:
Just as a comment but not to be included now, you may want to have a look at NPM logging packages such as https://www.npmjs.com/package/winston to ease the configuration of your logging needs (it's kind of the log4j package for logging in Node, one of them :) )